test/filters/test_functor_filter.cpp

```diff
@@ -90,28 +102,52 @@ class FunctorFilterExecutor : public ::testing::Test {
       return (pt.getArray3fMap() < 0).all();
     };
 
+    const auto compare_indices =
+        [](const auto first, const auto last, const index_t start) {
+          index_t i = start;
+          for (auto it = first; it != last; ++it, ++i)
+            ASSERT_EQ(*it, i);
```

Comment:
EXPECT instead of ASSERT?

Reply:
Is was to prevent the massive dump which occurs in the console in case the test fails. I have no strong preference on this.